doc/howto/storage_launch.md

```diff
@@ -0,0 +1,16 @@
+# How to launch an instance in a specific storage pool
```

Comment:
Can we call this "How to create an instance in a specific storage pool"? See https://github.com/lxc/lxd/pull/10673/files#r924196248

The file itself will need to be renamed also. Also can you include the word "instance" in the file name, as "storage_launch" isn't that descriptive of the actual contents of the file.